OpenSim/Simulation/Model/PhysicalFrame.cpp

```diff
@@ -48,7 +48,7 @@ PhysicalFrame::PhysicalFrame() : Frame()
     FrameGeometry frm(0.2);
```

Comment:
FrameGeometry should be removed and Frames should be able to draw/generate its own geometry.